code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -141,11 +141,13 @@ Made by Xhuis
 	var/mob/living/carbon/human/S = shadow_mind.current
 	shadow_mind.AddSpell(new /obj/effect/proc_holder/spell/self/shadowling_hatch(null))
 	shadow_mind.AddSpell(new /obj/effect/proc_holder/spell/self/shadowling_hivemind(null))
-	spawn(0)
-		update_shadow_icons_added(shadow_mind)
-		if(shadow_mind.assigned_role == "Clown")
-			to_chat(S, "<span class='notice'>Your alien nature has allowed you to overcome your clownishness.</span>")
-			S.dna.remove_mutation(CLOWNMUT)
+	update_shadow_icons_added(shadow_mind)
+	INVOKE_ASYNC(src, .proc/unclown, S, "Your alien nature has allowed you to overcome your clownishness.")
+
+/datum/game_mode/proc/unclown(mob/living/carbon/human/S, fluff_message)
+	if(shadow_mind.assigned_role == "Clown")
```

Comment:
We should probably move this if statement to the finalize_shadowling proc, that way we only add an extra timer when we need to, rather than for everyone that is a shadowling.